cdap-ui/app/features/flows/controllers/tabs/runs/runs-detail-ctrl.js

```diff
@@ -16,7 +20,11 @@ angular.module(PKG.name + '.feature.flows')
       event.targetScope.runs.selected = null;
     });
 
-    $scope.selectTab = function(tab) {
+    $scope.selectTab = function(tab, node) {
+      if (tab.title === 'Flowlets') {
```

Comment:
This is switching tabs. If we are switching to Flowlets, then we add the flowlet node name. Otherwise it can go to Logs and Status tab as well